SunEngine.Tests/DefaultInit.cs

```diff
@@ -0,0 +1,22 @@
+﻿using SunEngine.Core.DataBase;
+
+namespace SunEngine.Tests
+{
+    internal static class DefaultInit
+    {
+        private static string connectionString = "Host=localhost;Database=SunEngine;Username=postgres;Password=password";
+        private static string provider = "PostgreSQL.9.5";
+
+        public static DataBaseConnection GetTestDataBaseConnection()
+        {
+            LinqToDB.Common.Configuration.Linq.AllowMultipleQuery = true;
+            return new DataBaseConnection(provider, connectionString);
+        }
+
+        public static DataBaseFactory GetTestDataBaseFactory()
+        {
+            LinqToDB.Common.Configuration.Linq.AllowMultipleQuery = true;
+            return new DataBaseFactory(provider, connectionString, new DbMappingSchema());
```

Comment:
Where do we initialize database?

Reply:
Initialisation in constuctor on DataBaseFactory. Maybe we need overload constructor of DatabaseFactory?

Reply:
As I can see it creates connection, but DB itself is not created/populated there. 
Correct me if I'm wrong.